docs/rules/lines-around-comment.md

```diff
@@ -172,6 +172,101 @@ function foo(){
 }
 ```
 
+### allowClassStart
+
+Examples of **incorrect** code for this rule with the `{ "beforeLineComment": true, "allowClassStart": false }` option:
+
+```js
+/*eslint lines-around-comment: ["error", { "beforeLineComment": true, "allowClassStart": false }]*/
+
+class foo {
+    // what a great and wonderful day
+    day() {"great"}
```

Comment:
Can we all these examples say `day() {}` or `day() { return great; }` instead? It's a bit confusing as it currently stands.

Reply:
Switched them all to `day() {}` (and rebased)